docs/setup/config/README.md

```diff
@@ -0,0 +1,11 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Cartography Intel Module Configuration](#cartography-intel-module-configuration)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Cartography Intel Module Configuration
+
+Please view the other documents in this folder to learn how to configure each of your data sources for Cartography.
```

Comment:
We should link to the other docs here.

Reply:
I would, but I want to avoid needing to update the same docs in multiple places.  README.md already has a listing of the available modules and links to their config information.